notebook/static-src/base/js/namespace.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+"use strict";
+
+if (!window.Jupyter) {
+    var Jupyter = {};
+    Jupyter.version = "4.1.0.dev";
```

Comment:
When I say canonical though, I mean both from Python and from JavaScript. Which just means that we're updating and working from the same file.

We've been (understandably so), relying on the Pythonic way of embedding it in source under a python file (in our case `notebook/_version.py`).

I personally feel like we can use JSON even within `_version.py`, which... could mean pulling it from package.json. :smiling_imp: